tests/Fixtures/symlinks/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
I would add the directory to the root .gitignore file instead, with a comment to explain its purpose.